homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -139,16 +139,19 @@ def state(self):
     @property
     def sound_mode(self):
         """Return the current sound mode."""
-
         if self._equaliser == -1:
             return ""
+        if self._equaliser >= len(temescal.equalisers):
+            return ""
```

Comment:
This can be combined in a single statement

```suggestion
        if self._equaliser == -1 or self._equaliser >= len(temescal.equalisers):
            return ""
```

Reply:
Please `return None` on unknown value. Don't return empty string.